### ISSUE TITLE
how to add additional directories to pow-extra-dirs ?

### ISSUE THREAD

**Issue by kiuuik** [abo-abo/plain-org-wiki#4]
I changed line 40 from plain-org-wiki.el from `(defvar pow-extra-dirs nil` to `(defvar pow-extra-dirs "/home/user/additionalpath/"` and get this error message: 

    In ‘Projects’ source: ‘pow-files’ 
     (wrong-type-argument listp "~/home/user/additionalpath/")

I tried with the most recent version of your file plain-org-wiki.el and emacs 25.1/org9.0.4. Then I tried it with your configuration "oremacs" this afternoon. This throws another error in my virtualmachine with emacs 24.5: `pow-files: Symbol's function definition is void: mapcan`. So I put `(require 'cl)` at the beginning of plain-org-wiki.el. Then I get the same error as in my config (wrong-type-argument)

How do I add additional paths?

Unfortunately I don't know anything about emacs lisp. I only use org-mode because it's a good substitute for a simple wiki and it offers latex support and pdf export. 

I came to your tool because of this: I have my org files in four folders (like work, personal, sideproject). I still haven't found a good solution how to open files from these different locations. Some workarounds like org-projectile did have some unwanted side effects.

PS Thanks for your great software. hydra is indispensable for me. I use avy daily and org-download regularly.

**Comment by abo-abo** [comment]
See this commit: 88563c922346dfa817499e40f5ab56c72a7a04cc. It should have fixed the `mapcan` error. Make sure that you have the latest git version.

Try not to edit `plain-org-wiki.el`. Just put this into your config:

```elisp
(require 'plain-org-wiki)
(setq pow-directory "/home/user/path/")
(setq pow-extra-dirs '("/home/user/additionalpath-1/"
                       "/home/user/additionalpath-2/"))
```

**Comment by kiuuik** [comment]
Your solution works perfectly. Thanks a lot. 

About my mapcan error: I accidentally caused it when I wanted to circumvent a problem in emacs-oremacs (that I just downloaded with git clone): When I start it emacs freezes for a long time and I see this error in `*Messages*`: `Error (use-package): ispell :config: Symbol's function definition is void: ispell-hunspell-add-multi-dic`. I thought that maybe this problem is caused by missing dictionaries on the system and so I installed hunspell-nl and and hunspell-en-us (`sudo apt install hunspell-nl hunspell-en-us`). But the problem persisted. So I commented out the section "(use-package ispell ...)" (lines 245-252) in the init.el. Only then I got the mapcan error. It later occurred to me that it's enough to comment out the line `(ispell-hunspell-add-multi-dic "en_US,nl_NL")` ...

**Comment by abo-abo** [comment]
> When I start it emacs freezes for a long time and I see this error in *Messages*: Error (use-package): ispell :config: Symbol's function definition is void: ispell-hunspell-add-multi-dic

This is a built in function. It may have been added in Emacs 25, what's your Emacs version?

**Comment by kiuuik** [comment]
My last report was generated on ubuntu 16.10 with emacs 24.5.1. 
I tried in another vm with ubuntu 16.04 (without hunspell) with emacs 24.5.1. I get the same error message (`Error (use-package): ispell :config: Symbol's function definition is void: ispell-hunspell-add-multi-dic`) but emacs doesn't freeze.  Again the error disappears if I comment out line 252. 
I also tried in another VM with Fedora 25 which includes emacs 25.1.1. I get a different error message (and emacs starts as quickly as ususal): `Error (use-package): ispell :config: Wrong type argument: stringp, nil`. This error message disappears after I comment out line 251 `(ispell-set-spellchecker-params)`. Line 252 doesn't cause any error messages.
I always started with a clean oremacs folder and used "git clone", "make install", and "make run" this afternoon and night.
